components/shell/fish/Makefile

```diff
@@ -0,0 +1,50 @@
+#
```

Comment:
OK, i will do so. To generate the sample-manifest and p5m: is this the right document to study ? https://wiki.openindiana.org/oi/Building+an+IPS+manifest

This is my first contribution to oi-userland. I have to learn a lot.